app/js/components/common/notification/notification.component.js

```diff
@@ -21,17 +21,17 @@ const NotificationComponent = {
       };
 
       $rootScope.$on("TRIGGERED_NOTIFICATION", (evt, data) => {
-        const { message, type } = data;
+        const { message, type, isDismissable = false } = data;
         $log.log(`triggered: ${type} message=${message}`);
 
         vm.message = message;
         vm.type = type;
 
-        $scope.$apply();
-
-        $timeout(() => {
-          vm.removeNotification();
-        }, 5000);
+        if (!isDismissable) {
```

Comment:
Sorry, I'm going to get picky here. When I see `isDismissable`, I don't think of it automatically dismissing itself... The default here seems to be saying it cannot be dismissed (`isDismissable = false`) but then it's going ahead and auto-dismissing after 5 seconds. What about changing this `autoDismiss` with a default of `true`?